### DISCUSSION TITLE
ttkbootstrap in 3.9.4will not work

### DISCUSSION THREAD

**Discussion by nubbix** [israel-dryer/ttkbootstrap#486] [General]
I really love how this seems to work, but when trying this in VS Code using Python 3.9.4 on Windows. It does not work at all. 
Keep getting this error message while using the default examples from the website. 

So what or how do I fix this.  Even tried to fix it using ChatGPT which only refer me back to the developer of ttkbootstrap. 

Is it even possible to get to work?

Error message
`Traceback (most recent call last):
  File "c:\Users\post\Documents\Python Coding\Test\ttkbootstrap\app.py", line 1, in <module>
    import ttkbootstrap as ttk
  File "C:\Users\post\AppData\Local\Programs\Python\Python39\lib\site-packages\ttkbootstrap\__init__.py", line 3, in <module>
    from ttkbootstrap.widgets import *
  File "C:\Users\post\AppData\Local\Programs\Python\Python39\lib\site-packages\ttkbootstrap\widgets.py", line 14, in <module>
    from ttkbootstrap.dialogs import Querybox
  File "C:\Users\post\AppData\Local\Programs\Python\Python39\lib\site-packages\ttkbootstrap\dialogs\__init__.py", line 1, in <module>   
    from ttkbootstrap.dialogs.dialogs import *
  File "C:\Users\post\AppData\Local\Programs\Python\Python39\lib\site-packages\ttkbootstrap\dialogs\dialogs.py", line 536, in <module>  
    class DatePickerDialog:
  File "C:\Users\post\AppData\Local\Programs\Python\Python39\lib\site-packages\ttkbootstrap\dialogs\dialogs.py", line 566, in DatePickerDialog
    locale.setlocale(locale.LC_ALL, locale.setlocale(locale.LC_TIME, ""))
  File "C:\Users\post\AppData\Local\Programs\Python\Python39\lib\locale.py", line 610, in setlocale
    return _setlocale(category, locale)
locale.Error: unsupported locale setting`

**Comment by TheDemonGuard** [comment]
> I really love how this seems to work, but when trying this in VS Code using Python 3.9.4 on Windows. It does not work at all. Keep getting this error message while using the default examples from the website.
> 
> So what or how do I fix this. Even tried to fix it using ChatGPT which only refer me back to the developer of ttkbootstrap.
> 
> Is it even possible to get to work?
> 
> Error message `Traceback (most recent call last): File "c:\Users\post\Documents\Python Coding\Test\ttkbootstrap\app.py", line 1, in <module> import ttkbootstrap as ttk File "C:\Users\post\AppData\Local\Programs\Python\Python39\lib\site-packages\ttkbootstrap\__init__.py", line 3, in <module> from ttkbootstrap.widgets import * File "C:\Users\post\AppData\Local\Programs\Python\Python39\lib\site-packages\ttkbootstrap\widgets.py", line 14, in <module> from ttkbootstrap.dialogs import Querybox File "C:\Users\post\AppData\Local\Programs\Python\Python39\lib\site-packages\ttkbootstrap\dialogs\__init__.py", line 1, in <module> from ttkbootstrap.dialogs.dialogs import * File "C:\Users\post\AppData\Local\Programs\Python\Python39\lib\site-packages\ttkbootstrap\dialogs\dialogs.py", line 536, in <module> class DatePickerDialog: File "C:\Users\post\AppData\Local\Programs\Python\Python39\lib\site-packages\ttkbootstrap\dialogs\dialogs.py", line 566, in DatePickerDialog locale.setlocale(locale.LC_ALL, locale.setlocale(locale.LC_TIME, "")) File "C:\Users\post\AppData\Local\Programs\Python\Python39\lib\locale.py", line 610, in setlocale return _setlocale(category, locale) locale.Error: unsupported locale setting`

@nubbix 
Is it possible to retry with the latest ttkbootstrap version and check if the issue still persists?

In case the issue still persists,
Please provide working minimum reproducible demo/sample script.
And any necessary steps/instructions to reproduce the issue.
This will be helpful for anyone who will be interested in investigating the issue.
